### PR TITLE
chore(supernova): updates README and adds secretProps_sample

### PR DESCRIPTION
**Pull request by edda**
# Summary

This PR is an update to the Supernova Readme with some wording improvements. Also it adds a `secretProps_sample.js` for dev mode.

# Changes Made

- Updates Readme, especially the features section and the configuration section
- adds `secretProps_sample.js` with comments
- adds missing config props in `docker/appProps.js`

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by hodanoori — DISMISSED** [review]
lgtm

**Comment by andypf** [comment]
There is an issue with format in README

File: apps/supernova/README.md [hunk 1]
```diff
@@ -23,7 +29,7 @@ Alerting rules offer the capability to define alert conditions using expressions
 
 #### Labels
 
-The labels clause allows specifying a set of additional labels to be attached to the alert. Following is a live example of a set of labels from an alert of the `support group:containers` with `severity:info` in the `region:eu-de-2`.
+The labels clause allows specifying a set of additional labels to be attached to the alert. Following is a live example of a set of labels from an alert of the `support_group:containers` with `severity:info` in the `region:eu-de-2`.
```
Comment (ArtieReus) [inline apps/supernova/README.md:32]: Do we want to still mention the `support_group:containers` from the live example?

**Review by hodanoori — APPROVED** [review]
lgtm

**Review by ArtieReus — APPROVED** [review]
lgtm

**Pull request merged**